applications/migrations/0002_add_short_data_report.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 5.0.3 on 2024-03-21 15:50
+
+import django.db.models.deletion
+import django.utils.timezone
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("applications", "0001_initial"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="typeofstudypage",
+            name="is_study_short_data_report",
+            field=models.BooleanField(default=False),
+        ),
+        migrations.CreateModel(
```

Comment:
We may want this table to be available in grafana to query. If so, you'll need to give the grafana db user permission to query it: https://github.com/ebmdatalab/metrics/blob/6272e24320543401545275ccf946d89cad06a0fd/grafana/README.md?plain=1#L64C1-L67C13